v2/internal/controllers/recordings/Test_ApiManagement_20230501preview_CRUD.yaml

```diff
@@ -1,3098 +0,0 @@
----
```

Comment:
I don't think the deletion of this file was intentional?

If you're having trouble re-recordign this test, let us know and we can help.

Reply:
I am working on updating the recordings